django/core/checks/security/base.py

```diff
@@ -182,3 +187,9 @@ def check_xframe_deny(app_configs, **kwargs):
         settings.X_FRAME_OPTIONS == 'DENY'
     )
     return [] if passed_check else [W019]
+
+
+@register(Tags.security, deploy=True)
+def check_allowed_hosts(app_configs, **kwargs):
+    passed_check = not settings.ALLOWED_HOSTS
```

Comment:
`passed_check` is to check if list is not empty.
if it is not empty, method will return no error, otherwise error will be returned ([] if passed_check else [W020]).